src/main/java/com/thevoxelbox/voxelsniper/brush/type/SignOverwriteBrush.java

```diff
@@ -44,85 +42,132 @@ public SignOverwriteBrush(File pluginDataFolder) {
+                        messenger.sendMessage(ChatColor.BLUE + "Line " + firstParameter + " is " + ChatColor.GREEN +
+                                (this.signLinesEnabled[lineIndex] ? "enabled" : "disabled") + ChatColor.BLUE + ".");
+                    } else {
+                        messenger.sendMessage(ChatColor.RED + "Invalid brush parameters! Use the \"info\" parameter to display parameter info.");
                     }
-                    this.rangedMode = (parameters[++index].equalsIgnoreCase("on") || parameters[++index].equalsIgnoreCase("yes"));
-                    messenger.sendMessage(ChatColor.BLUE + String.format(
-                            "Ranged mode is %s",
-                            ChatColor.GREEN + (this.rangedMode ? "enabled" : "disabled")
-                    ));
+                } else if (firstParameter.equalsIgnoreCase("multiple") || firstParameter.equalsIgnoreCase("m")) {
+                    this.rangedMode = Boolean.parseBoolean(parameters[1]);
+                    messenger.sendMessage(ChatColor.BLUE + "Ranged mode is " + ChatColor.GREEN + (this.rangedMode ? "enabled" :
+                            "disabled") + ChatColor.BLUE + ".");
                     if (this.rangedMode) {
-                        messenger.sendMessage(ChatColor.GREEN + "Brush size set to " + ChatColor.RED + toolkitProperties.getBrushSize());
-                        messenger.sendMessage(ChatColor.AQUA + "Brush height set to " + ChatColor.RED + toolkitProperties.getVoxelHeight());
-                    }
-                } else if (parameter.equalsIgnoreCase("-save") || parameter.equalsIgnoreCase("-s")) {
-                    if ((index + 1) >= parameters.length) {
-                        messenger.sendMessage(ChatColor.RED + String.format("Missing parameter after %s.", parameter));
-                        continue;
+                        messenger.sendMessage(ChatColor.GREEN + "Brush size set to: " + ChatColor.RED + toolkitProperties.getBrushSize());
+                        messenger.sendMessage(ChatColor.AQUA + "Brush height set to: " + ChatColor.RED + toolkitProperties.getVoxelHeight());
                     }
-                    String fileName = parameters[++index];
+                } else if (firstParameter.equalsIgnoreCase("save") || firstParameter.equalsIgnoreCase("s")) {
+                    String fileName = parameters[1];
                     saveBufferToFile(snipe, fileName);
-                } else if (parameter.equalsIgnoreCase("-open") || parameter.equalsIgnoreCase("-o")) {
-                    if ((index + 1) >= parameters.length) {
-                        messenger.sendMessage(ChatColor.RED + String.format("Missing parameter after %s.", parameter));
-                        continue;
-                    }
-                    String fileName = parameters[++index];
+                } else if (firstParameter.equalsIgnoreCase("open") || firstParameter.equalsIgnoreCase("o")) {
```

Comment:
Attempting to load an invalid file sets the default?
![](https://i.imgur.com/1c15y4L.png)

Reply:
Buffer has always these values, however shouldn't be sent if file loading fails, ty.